### PR TITLE
Delete First page demarrer on enigma 

### PR DESCRIPTION
**Pull request by yudino**
### Motivation

First page demarrer delete on create an enigma and breadcrumbs clickable.

### Changes

In files "creer-une-enigme" 1 to 4.tsx

### Test

Got to "Créer une énigme" et choose one topic. You should see the changes.

### CODE REVIEW

**Review by DavidRobertAnsart — APPROVED** [review]
👍

File: src/pages/creer-une-enigme/4.tsx [hunk 1]
```diff
@@ -1,105 +1,195 @@
...
+                      router.push(`/creer-une-enigme/1?edit=${activity.id}`);
+                    }}
+                    status={'success'}
+                    style={{ position: 'absolute', top: '0.5rem', right: '0.5rem' }}
+                  />
+                )}
+                <Activities activities={[responseActivity]} noButtons />
+              </div>
+            </>
+          )}
```
Comment (DavidRobertAnsart) [inline src/pages/creer-une-enigme/4.tsx:140]: **Nit:** Maybe I didn't catch those in your previous PRs, but this would be great to do in another PR anyway. Can you remove all stuff related to `responseActivity` in these activities, now that there is a specfic one for that (Reaction activity). So we don't need that here.

**Review by DavidRobertAnsart — APPROVED** [review]
Lgtm!

File: src/pages/creer-une-enigme/4.tsx (hunk 1 above)
Comment (yudino) [reply src/pages/creer-une-enigme/4.tsx:140]: Oki I will do it in a global PR for this one.
